### PR TITLE
Update Snail people for reasons beyond comprehension

### PR DESCRIPTION
**Pull request by ghost**
Balances out snail people, giving them a speed(from 6 to 2, the same as golems) and unarmed damage boost(from 0.5 to 5 brute a hit) while lowering their natural armor and burn weakness, effectively making them playable.

## About The Pull Request

Changes snails to not be shit.

## Why It's Good For The Game

Because its something I said I was gonna do

## Changelog

Changed:
Burnmod from 2 to 1.5
Speedmod from 6 to 2 
Punchdamagehigh from 0.5 to 5
Revalued the shell backpacks armor so grey tiders weren't walking around in armored vests round start, but still give a a small improvement to deal with the speed and burn damage traits.

Edit: 

Removed their snailcrawl; When snails laid down, they would leave lubed up tiles behind them and get a speed boost
Removed their phobia of salt, making them inable to move for any reason over salt piles.
Removed their noslip trait

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Please write the changelog out

**Comment by qwertyquerty** [comment]
also you should have waited to add snail people so I could add them to the shop first

**Comment by ghost** [comment]
Okay I think I did it right?

**Comment by qwertyquerty** [comment]
Looks like you made this race kind of OP.

**Comment by ghost** [comment]
The armor is near negligible unless you're wearing additional armor, which even then won't boost it my a significant amount, the punch damage from 0.5 to 5 is more then reasonable given they literally would deal less then 1 brute a hit, the speed I was debating on lowering to a 3 until I relized how unbearable it would be for people to wear hardsuits with this race,.

All in all their just humans who take more burn damage, can't eat salt, walk slower, and when wearing armor are slightly more robust in defence.

**Comment by ghost** [comment]
That one there is to remove snailcrawl for now, and to remove snails having no slip because I forgot to do that

**Comment by ghost** [comment]
That one should fix their salt tile bullshit shinanigains

**Comment by CthulhuOnIce** [comment]
travis failed

**Comment by CthulhuOnIce** [comment]
```
code/modules/mob/living/carbon/human/species_types/snail.dm:36:error: incorrect number of macro arguments
code/modules/mob/living/carbon/human/species_types/snail.dm:42:error: incorrect number of macro arguments
```

File: code/modules/mob/living/carbon/human/species_types/snail.dm [hunk 1]
```diff
@@ -33,14 +33,13 @@
 	if(!istype(bag, /obj/item/storage/backpack/snail))
 		if(C.dropItemToGround(bag)) //returns TRUE even if its null
 			C.equip_to_slot_or_del(new /obj/item/storage/backpack/snail(C), SLOT_BACK)
-	C.AddComponent(/datum/component/snailcrawl)
-	ADD_TRAIT(C, TRAIT_NOSLIPALL, SPECIES_TRAIT)
+	ADD_TRAIT(C, SPECIES_TRAIT)
```
Comment (CthulhuOnIce) [inline code/modules/mob/living/carbon/human/species_types/snail.dm]: revert this

File: code/modules/mob/living/carbon/human/species_types/snail.dm [hunk 2]
```diff
@@ -33,14 +33,13 @@
...
 
 /datum/species/snail/on_species_loss(mob/living/carbon/C)
 	. = ..()
 	var/datum/component/CP = C.GetComponent(/datum/component/snailcrawl)
 	CP.RemoveComponent()
-	REMOVE_TRAIT(C, TRAIT_NOSLIPALL, SPECIES_TRAIT)
+	REMOVE_TRAIT(C, SPECIES_TRAIT)
```
Comment (CthulhuOnIce) [inline code/modules/mob/living/carbon/human/species_types/snail.dm]: Revert this

**Review by CthulhuOnIce — CHANGES_REQUESTED** [review]
revert some changes

**Pull request merged**